gallery/visualtests/testUtils.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { Target } = require('@applitools/eyes-webdriverio');
+
+module.exports = {
+  simpleTest(selector) {
+    return async () => {
+      const targetElement = await browser.$(selector);
+
+      await targetElement.scrollIntoView({ block: 'center' });
+      await browser.eyesRegionSnapshot(null, Target.region(targetElement));
+    };
+  },
+
+  hoverTest(elementSelector, hoverSelector = elementSelector) {
+    return async () => {
+      const [targetElement, hoverElement] = await Promise.all([browser.$(elementSelector), browser.$(hoverSelector)]);
+
+      await targetElement.scrollIntoView({ block: 'center' });
+      await hoverElement.moveTo();
+
+      await browser.saveScreenshot('/tmp/before.png');
```

Comment:
Where can I read about the difference between `browser.saveScreenshot` and `browser.eyesRegionSnapshot()`?

What do we do with the screenshots?

Reply:
> Where can I read about the difference between browser.saveScreenshot and browser.eyesRegionSnapshot()?

`saveScreenshot` saves the screenshot to a local file, and has nothing to do with applitools.  The calls to that were actually for debugging and I need to remove them; thanks for reminding me that they're still there.

`eyesRegionSnapshot` is one of two custom commands that I added to the `browser` object in `wdio.conf.js`.

Reply:
> The calls to that were actually for debugging and I need to remove them; thanks for reminding me that they're still there.

fb6f499c